Tests/AppTests/SearchTests.swift

```diff
@@ -52,8 +52,8 @@ class SearchTests: AppTestCase {
             filters: [try AuthorSearchFilter(expression: .init(operator: .is, value: "foo"))]
         )
 
-        _assertInlineSnapshot(matching: renderSQL(b), as: .lines, with: """
-              SELECT 'package' AS "match_type", NULL AS "keyword", "package_id", "package_name", "repo_name", "repo_owner", "score", "summary", "stars", "license", "last_commit_date", "last_activity_at" FROM "search", CONCAT("keywords") AS "keyword" WHERE CONCAT_WS(' ', "package_name", COALESCE("summary", ''), "repo_name", "repo_owner") ~* $1 AND "repo_owner" IS NOT NULL AND "repo_name" IS NOT NULL AND ("repo_owner" ILIKE $2) ORDER BY LOWER("package_name") = $3 DESC, "score" DESC, "package_name" ASC
+        XCTAssertEqual(renderSQL(b), """
```

Comment:
I've changed these back to `XCTAssertEqual` because with the recent Kaleidoscope update that lets you diff XCTest output it's actually now easier again to see changes _that_ way.